modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataDefinitionResourceTest.java

```diff
@@ -125,6 +125,15 @@ public void testDeleteDataDefinition() throws Exception {
 				parentDataDefinition.getId()));
 	}
 
+	@Override
+	@Test
+	public DataDefinition
+			testDeleteSiteDataDefinitionByContentTypeByExternalReferenceCode_addDataDefinition()
+		throws Exception {
+
+		return _addDataDefinition("data-definition-1.json");
```

Comment:
can you change this return for this:
```
return dataDefinitionResource.postSiteDataDefinitionByContentType(
			testGroup.getGroupId(), _CONTENT_TYPE, randomDataDefinition());
```

Reply:
See this example: `testPutDataDefinitionPermissionsPage_addDataDefinition()`

Reply:
Sure, np